app/views/application/sign_in.html.erb

```diff
@@ -0,0 +1,15 @@
+  <%= tag.h1 :Belépés,class:'ui header'%>
+
+<%= form_tag session_for_registered_path,class:'ui form' do %>
+   <%= label_tag :mail%>
+   <%= text_field_tag :mail %>
+   <br>
+   <%= label_tag :password%>
+   <%= password_field_tag :password%>
+   <br>
+   <%= submit_tag 'Belépés', class:'ui button'%>
+<%end%>
+
+<%= link_to 'AutSCH',oauth_login_path %>
+<br>
+<%= link_to 'Regisztráció',new_registration_path %>
```

Comment:
Ezek a linkek kapjanak 'ui button' class-t, és vigyük be a form-ba a Belépés gomb után (a br-t meg ki lehet törölni). Így szépen egymás mellé kerülnek.